### ISSUE TITLE
Media Player Classic playback cuts out audio inside confidential box

### ISSUE THREAD

**Issue by EnfLzSp** [sandboxie-plus/Sandboxie#4324]
### Describe what you noticed and did

- Media Player Classic x64 can stop playback when remaining idle and not repeat track playback
- Media Player Classic x64 starts with audio not working every once in a while
- Media Player Classic x64 does not start with DropAdminRights=y without Cryptorgraphic Service errors (which is probably related to a recently reported issue mentioned in the latest change log for not-yet released Sandboxie)

Steps to reproduce:
1. Use Sandboxie Classic 5.70
2. Install madVR test build 207 outside of Sandboxie (see download link in appropriate section below)
3. Configure Media Player Classic (MPC-HC64.exe) to use Sandboxie.exe provided on the bottom

### How often did you encounter it so far?

Once every 3 launches

### Expected behavior

Playback continues as usual

### Affected program

Media Player Classic x64, madVR

### Download link
https://github.com/clsid2/mpc-hc/releases/download/2.3.6/MPC-HC.2.3.6.21.x64.zip
https://www.videohelp.com/download/madVR09217.zip

### Where is the program located?

The program is installed both inside and outside the sandbox.

### Did the program or any related process close unexpectedly?

No, not at all.

### What version of Sandboxie are you running now?

5.70

### Is it a new installation of Sandboxie?

I recently did a new clean installation.

### In which sandbox type you have this problem?

In a security hardened sandbox (orange sandbox icon).

### Can you reproduce this problem on a new empty sandbox?

I can confirm it also on a new empty sandbox.

### What is your Windows edition and version?

Windows 24H2 26100.2152

### In which Windows account you have this problem?

A local account (Administrator).

### Please mention any installed security software

Not relevant

### Did you previously enable some security policy settings outside Sandboxie?

- Enabled flags for DEP, ATL Thunk, SEHOP, ASLR, Heap Termination on Corruption, Bottom Up ASLR, High Entropy ASLR, and Control Flow Guard, 
- Disabled Extension Point, remote loads, low integrity loads, non-system fonts.

### Sandboxie.ini configuration

```ini
Global:
SandboxieLogon=y
CoverBoxedWindows=y
OpenSamEndpoint=n
HideNonSystemProcesses=y
HideSbieProcesses=y
HideOtherBoxes=y
HideDiskSerialNumber=y

Box-Specific:
ForceProcess=madTPG.exe
ForceProcess=madHCCtrl.exe
ForceProcess=MPC-HC64.exe
DropAdminRights=madTGP.exe,y
DropAdminRights=madHCCtrl.exe,y
FakeAdminRights=MPC-HC64.exe,y
BlockScreenCapture=MPC-HC64.exe,y
UseSecurityMode=y
UsePrivacyMode=y
HideNetworkAdapterMAC=y
RestrictDevices=y
SysCallLockDown=y
UseRuleSpecificity=y
RandomRegUID=y
ConfidentialBox=y
DenyHostAccess=audiodg.exe,n
NotifyStartRunAccessDenied=n
ClosedFilePath=InternetAccessDevices
```

**Comment by EnfLzSp** [comment]
Yes, it does appear that out-of-focus is a factor in top 2 issues I mention.

**Comment by DavidXanatos** [comment]
@offhub if the template changes in 1.15.1 have fixed this then we can close the issue?

**Comment by DavidXanatos** [comment]
how to reproduce the crypto errors? open a https video stream or alike? demo link please

**Comment by offhub** [comment]
@DavidXanatos The title of this issue can be misleading. When videos are started playing while the media player is out of focus, they play without sound, and to fix this, `OpenWinClass=*` needs to be used. The only problem is that when a program is specified for this setting, `OpenWinClass=MPC-HC64.exe,*`, it doesn't produce the same result. The MMCSS setting added to the templates has no relevance to this issue

>how to reproduce the crypto errors? open a https video stream or alike? demo link please

This issue has already been fixed in version 1.15.1. https://github.com/sandboxie-plus/Sandboxie/blob/f97cce1f6c03a3bdd9c275c44ce9b1e9d6e52bdc/CHANGELOG.md?plain=1#L35
